MSAL/src/configuration/external/ios/MSALLegacySharedADALAccount.m

```diff
@@ -132,7 +132,7 @@ - (instancetype)initWithJSONDictionary:(NSDictionary *)jsonDictionary error:(NSE
         _username = [jsonDictionary msidStringObjectForKey:@"username"];
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->